docs/layouts/shortcodes/generated/core_configuration.html

```diff
@@ -473,6 +473,18 @@
             <td>String</td>
             <td>The time zone to parse the long watermark value to TIMESTAMP value. The default value is 'UTC', which means the watermark is defined on TIMESTAMP column or not defined. If the watermark is defined on TIMESTAMP_LTZ column, the time zone of watermark is user configured time zone, the value should be the user configured local time zone. The option value is either a full name such as 'America/Los_Angeles', or a custom timezone id such as 'GMT-08:00'.</td>
         </tr>
+        <tr>
+            <td><h5>snapshot.expire.execution-mode</h5></td>
```

Comment:
Why you need to modify this?

Reply:
The change is generated automatically by "mvn package -Pgenerate-docs -pl paimon-docs -nsu", I will remove it.